futures-channel/benches/sync_mpsc.rs

```diff
@@ -27,20 +27,20 @@ fn unbounded_1_tx(b: &mut Bencher) {
         for i in 0..1000 {
 
             // Poll, not ready, park
-            assert_eq!(Poll::Pending, rx.poll_next_unpin(lw));
+            assert_eq!(Poll::Pending, rx.poll_next_unpin(waker));
```

Comment:
nit: here and elsewhere, I think it'd be fine to abbreviate waker as e.g. `wk` since it's used so often. I can open this as a separate followup change, though, since it seems potentially controversial.